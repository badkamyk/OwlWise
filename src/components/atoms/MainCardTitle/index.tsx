export default function MainCardTitle({ text }: { text: string }) {
	return <p className="text-lg font-semibold text-slate-800">{text}</p>;
}
