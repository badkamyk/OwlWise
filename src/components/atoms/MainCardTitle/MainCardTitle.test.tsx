import { render, screen } from "@testing-library/react";
import { expect, it } from "vitest";
import MainCardTitle from "./index";

it("MainCardTitle with props", () => {
	const { container } = render(<MainCardTitle text="Hello World" />);
	expect(screen.getByText("Hello World")).toBeTruthy();
	expect(
		container.getElementsByClassName("text-lg font-semibold text-slate-800")
	).toBeTruthy();
	expect(container).toMatchSnapshot();
});
