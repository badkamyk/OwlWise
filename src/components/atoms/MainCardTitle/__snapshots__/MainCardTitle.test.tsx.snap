// Vitest Snapshot v1

exports[`renders MainCardTitle with props correctly 1`] = `
<div>
  <p
    class="text-lg font-semibold text-slate-800"
  >
    Hello World
  </p>
</div>
`;
